src/start/CompatibilityScreen.js

```diff
@@ -15,17 +16,25 @@ const styles = StyleSheet.create({
     color: 'white',
     fontSize: 20,
     margin: 8,
-  }
+  },
 });
 
 export default class CompatibilityScreen extends React.Component {
-
+  handleClick = (link: string) => {
+    Linking.canOpenURL(link).then(supported => {
+      if (!supported) {
+        Alert.alert('Loading error', 'Could not open the store.');
```

Comment:
Maybe change this to 'Please go to the app store to download the latest version'?